app/routes/user/email/index.tsx

```diff
@@ -70,7 +96,55 @@ export default function () {
         </a>
         .
       </p>
-      {data && (
+      <h2>GCN Circulars</h2>
+      <p>
+        Subscriptions to <strong>Circulars</strong> are sent from GCN Circulars{' '}
+        {`<no-reply@${hostname}>`} and will be delivered to the email associated
+        with this account.
```

Comment:
Add the user's email address here.

```suggestion
        with your account ({email}).
```